### PR TITLE
create harness automation

### PR DESCRIPTION
**Pull request by nodish**

### CODE REVIEW

**Comment by nodish** [comment]
I signed it!

File: tools/harness-automation/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+=======================
+Harness Automation Tool
+=======================
+
+This is a tool to automate testing openthread with GRL thread harness.
```
Comment (jwhui) [inline tools/harness-automation/README.rst]: Indicate the exact version of the Thread harness that this tool was tested against.

File: tools/harness-automation/doc/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
+Thread Harness Automation Quick Start
+=====================================
+
+Setup
+-----
+
+#. Install Latest Thread Harness
```
Comment (jwhui) [inline tools/harness-automation/doc/quickstart.rst]: Indicate the exact version of the Thread harness that this tool was tested against.

File: tools/harness-automation/doc/quickstart.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+#. Install python 2.7
+#. Get the code
```
Comment (jwhui) [inline tools/harness-automation/doc/quickstart.rst]: Provide example command line here (for example `git clone ...`).

**Comment by jwhui** [comment]
Thanks for contributing this!  Just a few minor comments for the documentation.

**Comment by nodish** [comment]
Updated.

**Pull request closed without merging**